controllers/userControllers.js

```diff
@@ -0,0 +1,185 @@
+const model = require('../models')
+const comparePassword = require('../helpers/comparePassword')
+const jwt = require('jsonwebtoken')
+const ENV = require('dotenv')
+ENV.config()
+
+class UserController {
+  static createUser(req, res) {
+    const { firstName, lastName, email, username, password, role } = req.body
+
+    model.User
+      .create({
+        firstName: firstName,
```

Comment:
Sebenernya ketika kita ingin membuat sebuah object yang key dan nama variable dari valuenya sama ada shorthand yang bisa digunakan

```suggestion
        firstName, // sama dengan firstName: firstName
```